### PR TITLE
Jenkins Failing Test Fix(ONEPLAT-1702)

### PR DESCRIPTION
**Pull request by mudit94**
# Closes/Fixes/Resolves

fix for search test failing cases and some optimzations(adding timeouts)

# Explain the feature/fix

## Screenshots
feedback2
api
ssi
ug
idea
lh

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: e2e/integration/searchtests.js [hunk 1]
```diff
@@ -81,14 +78,17 @@ context( 'Test search', () => {
...
-        cy.get( 'opc-nav-search[slot="opc-nav-search"]' ).within( () => {
-            cy.get( '.opc-nav-search__btn' ).click( { force: true } );
-            cy.wait( 500 );
-            cy.get( 'input[name="query"]' ).click( { force: true } ).clear( { force: true } ).type( 'qwertyuiopasdgthchgbtnjkmnvxs', { force: true } );
-            cy.get( '.opc-nav-search__btn' ).click( { force: true } );
-        } )
+        cy.wait( 15000 );
+        cy.get('.opc-nav-search__btn', { timeout: 5000 } ).click( { force: true } );
+        cy.wait(3000)
+        cy.get( 'input[name="query"]',{timeout:5000} ).focus().type( 'qwertyuiopasdgthchgbtnjkmnvxs', { force: true } );
```
Comment (hybridx) [inline e2e/integration/searchtests.js:88]: Is there a specific reason to give a random string @mudit94 ?
I guess you can add something which is relevant.

File: e2e/integration/apicatalogtests.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ Cypress.on( 'uncaught:exception', ( err, runnable ) => {
 } )
 context( 'API Catalog Tests', () => {
     before( () => {
+        cy.viewport( 1280, 720 );
```
Comment (hybridx) [inline e2e/integration/apicatalogtests.js:9]: @mudit94 if all the viewports have the same value maybe using a variable would make sense right?

File: package.json [hunk 3]
```diff
@@ -56,7 +56,7 @@
     "@typescript-eslint/eslint-plugin": "4.32.0",
     "@typescript-eslint/parser": "4.32.0",
     "bats": "1.4.1",
-    "cypress": "6.5.0",
+    "cypress": "7.6.0",
```
Comment (hybridx) [inline package.json:59]: @mudit94 can you please also update the `package-lock.json` ?

**Review by hybridx — CHANGES_REQUESTED** [review]

File: e2e/integration/searchtests.js (hunk 1 above)
Comment (mudit94) [reply e2e/integration/searchtests.js:88]: Hi @hybridx , random string is given to check for invalid search response

File: e2e/integration/apicatalogtests.js (hunk 2 above)
Comment (mudit94) [reply e2e/integration/apicatalogtests.js:9]: Hi @hybridx , viewports are basically browser window size which I configured to make it maximize

Comment (hybridx) [reply e2e/integration/apicatalogtests.js:9]: @riginoommen can you please double check with @mudit94 about using a single variable across different files

**Review by hybridx — APPROVED** [review]
lgtm

File: e2e/integration/apicatalogtests.js (hunk 2 above)
Comment (mudit94) [reply e2e/integration/apicatalogtests.js]: hi @hybridx @riginoommen fixed in https://github.com/1-Platform/one-platform/pull/1355/commits/1a3e42f2a6e43249661271f159a25098d88d884b

**Comment by amalkmathew** [comment]
This changes looks good to me can you close https://github.com/1-Platform/one-platform/pull/1324 PR if this pr solved all issues

**Review by riginoommen — APPROVED** [review]

File: e2e/integration/innovationhubtests.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 context( 'Test for Innovation Hub', () => {
```
Comment (riginoommen) [inline e2e/integration/innovationhubtests.js:1]: Rename this file to ideahub tests

Comment (mudit94) [reply e2e/integration/innovationhubtests.js:1]: https://github.com/1-Platform/one-platform/pull/1355/commits/0027b659f74abff15b59f979f7ff3ad6d8b201c6 done @riginoommen

File: package.json (hunk 3 above)
Comment (riginoommen) [reply package.json:59]: I see its updated

**Pull request merged**